docs/notebooks/analysis/fit_functions.ipynb

```diff
@@ -0,0 +1,380 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Fit Functions\n",
+    "\n",
+    "[Fit functions]: ../../api_static/plasmapy.analysis.fit_functions.rst\n",
+    "[linregress()]: https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.linregress.html\n",
+    "[curve_fit()]: https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.curve_fit.html\n",
+    "[fsolve()]: https://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.fsolve.html\n",
+    "\n",
+    "**[Fit functions]** are a set of callable classes designed to aid in fitting analytical functions to experimental data.  A fit function class combines the following functionality into one package:\n",
```

Comment:
```suggestion
    "**[Fit functions]** are a set of callable classes designed to aid in fitting analytical functions to experimental data.  A fit function class combines the following functionality:\n",
```
The word "package" could be ambiguous here, since it's often referred to a whole Python package.